lib/src/http/managers/channel_manager.dart

```diff
@@ -825,4 +825,27 @@ class ChannelManager extends ReadOnlyManager<Channel> {
 
     stageInstanceCache.remove(channelId);
   }
+
+  Future<void> addRecipient(Snowflake channelId, Snowflake userId, {required String accessToken, required String nick}) async {
```

Comment:
These named parameters should be moved to a builder - maybe `DmRecipientBuilder`?